### PR TITLE
Add SCIM API page

### PR DESCRIPTION
**Pull request by anle-statsig**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://673bb8303d5e303d92bd7fdf--cozy-fox-0defec.netlify.app

File: src/components/Rapidoc.js [hunk 1]
```diff
@@ -24,28 +23,6 @@ const entityToTagMap = {
...
-      }
-    });
-  });
-
-  return {
-    ...spec,
-    paths: filteredPaths,
-  };
-}
-
```
Comment (jairo-statsig) [inline src/components/Rapidoc.js:48]: Why was this moved?

**Review by jairo-statsig — APPROVED** [review]

File: src/components/Rapidoc.js (hunk 1 above)
Comment (anle-statsig) [reply src/components/Rapidoc.js:48]: I cleaned this file up. It's quite messy

**Comment by anle-statsig** [comment]
### Merge activity

* **Nov 18, 8:46 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/statsig-io/docs/2352).
